contrib/drf_introspection/serializers.py

```diff
@@ -22,17 +24,51 @@ def _error_with_fields(message, fields):
 
 def _verify_field_names(fields, valid_fields, filter_name):
     if fields:
-        invalid_field_names = set(fields) - valid_fields
+        invalid_field_names = fields - valid_fields
         if invalid_field_names:
             error = 'Unknown fields in "%s" filter: %s' % (filter_name, ', '.join(invalid_field_names))
             raise FieldError(error)
 
 
-def _pop_field_list(args, argument_name):
-    values = args.pop(argument_name, [])
+def _normalized_fields_set(values):
+    """
+    Returns a set for `fields` and `exclude_fields` arguments.
+
+    >>> _normalized_fields_set("a")
+    set(['a'])
+    >>> _normalized_fields_set(["a"])
+    set(['a'])
+    >>> _normalized_fields_set(["a", "b"]) == set(['a', 'b'])
+    True
+
+    >>> _normalized_fields_set("a,b") == set(['a', 'b'])
+    True
+    >>> _normalized_fields_set(["a,b"]) == set(['a', 'b'])
+    True
+    >>> _normalized_fields_set(["a,b", "c"]) == set(['a', 'b', 'c'])
```

Comment:
`doctest` is pretty cool for lazy people but it expects the exact output (e.g. `set(['a', 'c', 'b'])` instead of `set(['a', 'b', 'c'])`). Not sure if there is way around it.

Tested with command:
```bash
DJANGO_SETTINGS_MODULE=pdc.test_settings python -m doctest -v contrib/drf_introspection/serializers.py
```